test_graphs.py

```diff
@@ -33,4 +37,7 @@ def pytest_train_model(model_type):
     error, true_values, predicted_values = test_trained_model(tmp_file, model_type)
 
     for true_value, predicted_value in zip(true_values, predicted_values):
-        assert abs(true_value[0] - predicted_value[0]) < 1e-3
+        if model_type == "PIN" or model_type == "MFC":
+            assert abs(true_value[0] - predicted_value[0]) < 0.1
+        else:
+            assert abs(true_value[0] - predicted_value[0]) < 0.5
```

Comment:
Is there a reason that different error thresholds are used for different models?

Reply:
See comments at the top of the description of the PR